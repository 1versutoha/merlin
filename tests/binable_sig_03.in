["find", "use", "bin_prot"]
["tell", "source", "module T : sig type ('a,'b) t with bin_io end = struct type ('a,'b) t = T of 'a * 'b with bin_io end\nmodule T1 : Bin_prot.Binable.S2 = T"]
["tell", "end"]
["errors"]
