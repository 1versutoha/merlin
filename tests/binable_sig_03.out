["return",{"result":true}]
["return",null]
["return",{"line":2,"col":35}]
["return",[]]
